angelsbioprocessing/locale/en/bio-processing.cfg

```diff
@@ -556,3 +578,50 @@ desert-garden-b=Advanced desert seed extraction
 
 swamp-garden-a=Swamp seed extraction
 swamp-garden-b=Advanced swamp seed extraction
+
+garden-cultivating=A low chance to grow gardens that you do not have yet. Not suitable for garden duplication.
```

Comment:
Don't have to make it obvious, leave the part out that you can't use it for duplication

Reply:
If you want people to figure out how to use the recipes for themselves, it is probably best not to have any recipe descriptions whatsoever. I will remove all recipe descriptions that I added, but also the ones that were present before – which were only exact copies of the recipe names.